arviz/tests/test_plots.py

```diff
@@ -21,12 +21,15 @@ class Models:
         models = load_cached_models(draws=500, chains=2)
         pymc3_model, pymc3_fit = models['pymc3']
         stan_model, stan_fit = models['pystan']
+    return Models()
+
 
+@pytest.fixture(scope='function')
```

Comment:
Using inline comments to be more targeted; you should be able to set this to be an autouse fixture here